src/main/java/com/estsoft/guesshangeul/post/controller/QuizPostController.java

```diff
@@ -77,4 +79,12 @@ public ResponseEntity<Void> deleteQuizPost(@PathVariable Long quizBoardId, @Path
 		quizPostService.deleteQuizPost(quizBoardId, id);
 		return ResponseEntity.status(HttpStatus.NO_CONTENT).build();
 	}
+
+	// 퀴즈 게시글 삭제
+	@Transactional
+	@DeleteMapping
+	public ResponseEntity<Void> deleteByPostIds(@PathVariable Long quizBoardId, @RequestParam List<Long> postId) {
```

Comment:
보통 REST하게 하면
```suggestion
@DeleteMapping
public ResponseEntity<Void> deleteByPostIds(@PathVariable Long quizBoardId, @PathVariable Long postId) {
```
로 단일 개수를 제거하는 경향이 있습니다.
그런데 List로 받는다면 RequestParam으로 잘 받아들여지는지 헷갈리네요.
아니면 RequestBody로 깔끔하게 받는 방법도 있을 것 같습니다.